week1/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>1st Seminar practice</title>
+  </head>
+  <body>
+    <div style="display: flex; flex-direction: column; align-items: center">
```

Comment:
```suggestion
    <div>
```
css 아직 하지도 않았는데 개고수시네요.